packages/features/shell/Shell.tsx

```diff
@@ -215,15 +215,17 @@ const useBanners = () => {
 const Layout = (props: LayoutProps) => {
   const banners = useBanners();
 
-  const showIntercom = localStorage.getItem("showIntercom");
+  const { data: user } = trpc.viewer.me.useQuery();
```

Comment:
can you not user useSession here?

Reply:
the main reason to use this here is because in useIntercom this api is called and the data is not available initially when boot() is called [LINK](https://github.com/SomayChauhan/cal.com/blob/33b9e8f48c8477a274d946123cf661e30143c57b/packages/features/ee/support/lib/intercom/useIntercom.ts#L26)